views/artist-search-results.hbs

```diff
@@ -0,0 +1,3 @@
+<h1>{{data.body.artists}}</h1>
+<img src="{{this}}" alt="artists.image">
+<button><a href="/albums/someArtistIdGoesHere">View Albums</a></button>
```

Comment:
You are sending an array of objects into the view, which means you need a loop here to iterate over it. 

If information that you are sending looks like this:
```res.render("artist-search-results", data.body.artists.items)```

It means it has no name --> so we must use keyword this to access information:

```
{{#each this}}
    <h1>{{this.name}}</h1>
    <img src="{{this.images.2.url}}" alt={{this.name}}>
    <button><a href="/albums/someArtistIdGoesHere">View Albums</a></button>
{{/each}}
```